backend/src/actions/grafana/createGrafanaDashboardObject.ts

```diff
@@ -1,15 +1,17 @@
 import createGrafanaPanelObject from './createGrafanaPanelObject';
 import { GrafanaDashboard, GrafanaDatasource } from '../../types';
+import createPromQLQueries from './createPromQLQueries';
 
 export default async function createGrafanaDashboardObject(
```

Comment:
From what I see, all the code in this function is synchronous, so it does not need to be an async function. It can simply return an object with type GrafanaDashboard instead of a Promise<GrafanaDashboard>

Reply:
Good catch! I used to have a Grafana API request in here but I have moved it and this is no longer necessary.